zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -90,31 +92,25 @@ class Handlers {
   }
 
   def handlePublic(
-    resourceDirs: Set[String],
     typesMap: Map[String, String]) =
     new Service[Request, Renderer] {
       private[this] var rendererCache = Map.empty[String, Future[Renderer]]
 
-      private[this] def getStream(path: String): Option[InputStream] = {
-        val resource = getClass.getResource(path)
-        if (resource == null || new File(resource.getPath).isDirectory) {
-          None
-        } else {
-          Option(resource.openStream())
-        }
-      }
+      private[this] def getResource(path: String): Option[URL] =
+        Option(getClass.getResource(path match {
+          case _ if path.split("/").last.contains(".") => s"/static$path"
```

Comment:
guessing `// if the path includes a file name (ex index.html), serve it relatively from static`

and the otherwise case is.. `// if there was no file name in the path, boot them to the home page`